src/megaapi_impl.cpp

```diff
@@ -17364,6 +17364,7 @@ void MegaApiImpl::sendPendingRequests()
 			bool transactions = (numDetails & 0x08) != 0;
 			bool purchases = (numDetails & 0x10) != 0;
 			bool sessions = (numDetails & 0x20) != 0;
+            request->setNumber(numDetails); // allow client to know which flags were used
```

Comment:
This one is missing in docstrings in `getExtendedAccountDetails`, `getAccountDetails` and `getSpecificAccountDetails`. Also we might want to use two integers for the avobe concept (flagsRequestedAccountDetails) and the one below (numberOfAccountDetailsToRetrieve)

Reply:
I see this one has been adjusted already, nice one, thanks!